edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/ModuleSet.cs

```diff
@@ -38,17 +38,6 @@ public ModuleSet(IImmutableDictionary<string, IModule> modules)
 
         public bool TryGetModule(string key, out IModule module) => this.Modules.TryGetValue(key, out module);
 
-        public ModuleSet ApplyDiff(Diff diff)
```

Comment:
Does this method no longer make sense?

Reply:
No, this was used originally to apply the deployment update from the twin patch. That is done in EdgeAgentConnection now, with DeploymentConfig.